packages/api/wrangler.toml

```diff
@@ -131,6 +131,14 @@ vars = { CLUSTER_API_URL = "https://alexandra-cluster-api-web3-storage.loca.lt",
 [env.alexandra.durable_objects]
 bindings = [{ name = "NAME_ROOM", class_name = "NameRoom0" }]
 
+[env.josh]
+workers_dev = true
+account_id = "b649c9389ef9c1d318ac2945b2415046"
+vars = { CLUSTER_API_URL = "https://josh-cluster-api-web3-storage.loca.lt", PG_REST_URL = "https://josh-postgres-api-web3-storage.loca.lt", ENV = "dev" }
```

Comment:
Would you mind adding the `GATEWAY_URL` as well?

Reply:
👍  Done, should I also add `GATEWAY_URL = "https://ipfs.io" ` to the template at the bottom of this file? Or is it only required for some configs?

Reply:
Good catch, yes please 👍

Reply:
Cool, added now, thanks!